profiles/scraper.py

```diff
@@ -202,6 +204,28 @@ class LinkedInScraper(SeleniumScraper):
 
     def login(self):
         self.visit(self.urls['LINKEDIN_LOGIN_URL'])
+        time.sleep(5)
+        self.get_by_xpath('USERNAME').send_keys(LINKEDIN_EMAIL)
+        self.get_by_xpath('PASSWORD').send_keys(LINKEDIN_PASSWORD + '\n')
+
+    def check_invalid_profile(self, profile_url):
+        self.visit(profile_url)
+        time.sleep(2)
+        if self.d.current_url != profile_url:
+            return False
+        return True
```

Comment:
2 seconds is too long to wait for, need to poll at millisecond intervals, this function shouldn't impact the performace of the get_profile_by_url drastically.

Reply:
Done, changed this to 0.5 second